### PR TITLE
increase limit on email send and email template save for assign/remind/revoke

### PR DESCRIPTION
**Pull request by muhammad-ammar**
**Description:** Increase limit on offer assignment email send and save template.

**JIRA:** [ENT-2708](https://openedx.atlassian.net/browse/ENT-2708)

### CODE REVIEW

File: src/components/CodeAssignmentModal/index.jsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ class CodeAssignmentModal extends React.Component {
   constructor(props) {
     super(props);
 
+    this.maxTemplateFieldLimit = 50000;
```
Comment (zamanafzal) [inline src/components/CodeAssignmentModal/index.jsx:22]: I think we should keep following in the utils and then import them in the components. `maxTemplateFieldLimit`, `templateErrorMessages` and 
```
Object.entries(templateErrorMessages).forEach(([key, message]) => {
      if (formData[key] && formData[key].length > this.maxTemplateFieldLimit) {
        errors[key] = message;
        errors._error.push(message);
      }
    });

```

Comment (muhammad-ammar) [reply src/components/CodeAssignmentModal/index.jsx:22]: I thought about this but didn't found a location where I can place the util. Also I am not sure how we will handle the above mentioned `Object.entries` in the util function? Should we pass the `errors`  object to util function?

Comment (zamanafzal) [reply src/components/CodeAssignmentModal/index.jsx:22]: Yeah, I think we should pass errors to this util function. 
Should we create a general constants file for this kind(maxTemplateFieldLimit) of entries?

Comment (muhammad-ammar) [reply src/components/CodeAssignmentModal/index.jsx:22]: Can you suggest where we can create a utils/constants file?

Comment (zamanafzal) [reply src/components/CodeAssignmentModal/index.jsx:22]: I think about this path
https://github.com/edx/frontend-app-admin-portal/tree/master/src/data/constants/

Comment (muhammad-ammar) [reply src/components/CodeAssignmentModal/index.jsx:22]: But this path is for only redux action based constants.

Comment (zamanafzal) [reply src/components/CodeAssignmentModal/index.jsx:22]: Then can we create it here.
https://github.com/edx/frontend-app-admin-portal/tree/master/src

Comment (muhammad-ammar) [reply src/components/CodeAssignmentModal/index.jsx]: @zamanafzal @sameenfatima78 I have refactored the code and moved validation logic to utils. Please check.

**Pull request merged**